CMakeLists.txt

```diff
@@ -16,6 +16,11 @@ endif (APPLE)
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/)
 
+if (ANDROID)
+    set(CMAKE_FIND_ROOT_PATH ${OPENMW_DEPENDENCIES_DIR} "${CMAKE_FIND_ROOT_PATH}")
```

Comment:
We already have the CMAKE_FIND_ROOT_PATH set on line 160. No need to set it to the same value twice.